hydromt_delft3dfm/dflowfm.py

```diff
@@ -1287,9 +1304,25 @@ def _setup_crosssections(
             # set crsloc and crsdef attributes to crosssections
             self.logger.info(f"Preparing 1D point crossections from {crosssections_fn}")
             gdf_cs = workflows.set_point_crosssections(
-                branches, gdf_cs, maxdist=self._network_snap_offset
+                branches, gdf_cs, maxdist=maxdist
+            )
+        elif crosssections_type == "special":  # read from the output
```

Comment:
I thought cross-sections type could only be "branch", "xyz", "point". What is special? SHould it be added to the docstrings of the different functions?
Additionally could this part be tested?

Reply:
I checked with Xiaohan and the 'special' x-section can indeed be removed